source/features/unreleased-commits.tsx

```diff
@@ -74,12 +75,15 @@ async function add(branchSelectorParent: HTMLDetailsElement): Promise<void> {
 		= aheadBy === undeterminableAheadBy
 			? 'more than 20 unreleased commits'
 			: pluralize(aheadBy, '$$ unreleased commit');
-	const label = `There are ${commitCount} since ${latestTag}`;
+	const label = `There are ${commitCount} since ${abbreviateString(latestTag, 30)}`;
```

Comment:
Instead of manually abbreviating we can use GitHub's own "truncate" class. They used it every time a branch name is used too

Reply:
AFAIK, CSS Truncate `text-overflow: ellipsis` only can be applied to `width fixed elements`
GitHub is using `Truncate` `Truncate-text` or `css-truncate` classes on their `inline elements` not tooltips.

Reply:
This is the `branch` link in PR headers, you can definitely use it here too. You just need two elements and the class already has a max-width

<img width="678" alt="Screenshot" src="https://github.com/refined-github/refined-github/assets/1402241/b50bbc89-5c2a-45b3-bde8-de87edeeae4c">

<img width="636" alt="Screenshot 4" src="https://github.com/refined-github/refined-github/assets/1402241/62424414-37aa-4da5-8586-26e061c0f3ef">

Reply:
🤔 I dont know is this possible.
The tooltip is `pseudo-element` and it effected by parent element's overflow attribute.
When we use `overflow: hidden` with `css-truncate > css-truncate-target` will not appears tooltip.
And the `text-overflow: ellipsis` needs `overflow: hidden`
![화면 기록 2023-09-10 20 22 28](https://github.com/refined-github/refined-github/assets/50487467/1862260a-3f2c-4654-9eeb-c423e792a713)

Reply:
Ahhh you're absolutely right.